### PR TITLE
docs(breaking-changes): typos and markdownlint

### PR DESCRIPTION
**Pull request by EdJoPaTo**

### CODE REVIEW

File: BREAKING-CHANGES.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Breaking Changes
 
 This document contains a list of breaking changes in each version and some notes to help migrate
-between versions. It is compiled manually from the commit history and changelog. We also tag PRs on
-github with a [breaking change] label.
+between versions. It is compiled manually from the commit history and change log. We also tag PRs on
```
Comment (orhun) [inline BREAKING-CHANGES.md:4]: We use `changelog` in some other places. I would prefer we revert this change to keep things consistent.

Comment (EdJoPaTo) [reply BREAKING-CHANGES.md:4]:
```suggestion
between versions. It is compiled manually from the commit history and changelog. We also tag PRs on
```

File: BREAKING-CHANGES.md [hunk 2]
```diff
@@ -385,13 +385,13 @@ The MSRV of ratatui is now 1.67 due to an MSRV update in a dependency (`time`).
...
+### `bitflags` updated to 2.3 ([#205])
 
 [#205]: https://github.com/ratatui-org/ratatui/issues/205
 
-The serde representation of bitflags has changed. Any existing serialized types that have Borders or
-Modifiers will need to be re-serialized. This is documented in the [bitflags
-changelog](https://github.com/bitflags/bitflags/blob/main/CHANGELOG.md#200-rc2)..
+The `serde` representation of `bitflags` has changed. Any existing serialized types that have Borders or
+Modifiers will need to be re-serialized. This is documented in the [`bitflags`
+change log](https://github.com/bitflags/bitflags/blob/main/CHANGELOG.md#200-rc2)..
```
Comment (joshka) [inline BREAKING-CHANGES.md:394]: Let's add this to the typos dictionary - it's reasonable to use this as a correct word.

**Review by joshka — APPROVED** [review]
LGTM, 1 nit

File: BREAKING-CHANGES.md (hunk 2 above)
Comment (EdJoPaTo) [reply BREAKING-CHANGES.md:394]: So… 'changelog' or 'change log'?

Comment (joshka) [reply BREAKING-CHANGES.md:394]: "changelog"

Comment (EdJoPaTo) [reply BREAKING-CHANGES.md:394]: The tool `typos` is not annoyed by it. LanguageTool is. → No change in this repo required.

Comment (EdJoPaTo) [reply BREAKING-CHANGES.md:394]:
```suggestion
changelog](https://github.com/bitflags/bitflags/blob/main/CHANGELOG.md#200-rc2)..
```

**Comment by joshka** [comment]
LGTM - lint error is a false positive typo in a test
